tldr.py

```diff
@@ -504,6 +508,40 @@ def main() -> None:
                 with open(command, encoding='utf-8') as open_file:
                     output(open_file.read().encode('utf-8').splitlines(),
                            plain=options.markdown)
+    elif options.search:
+        command = '-'.join(options.command)
+        get_commands(options.platform)
```

Comment:
delete?

Reply:
Updating this from gitter, but I'm not sure what the purpose of this lone `get_commands(options.platform)` here is, given that this same function is called 4 lines down to start the for loop. I think we can delete this line?

```suggestion
```

Reply:
Yea you're right, this serves no real purpose here